e2e/cypress/support/index.ts

```diff
@@ -15,19 +15,64 @@
 
 // Import commands.js using ES2015 syntax:
 import './commands';
+import '../integration/constants';
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
 
+const IAMV2 = 'v2.1';
+const IAMV1 = 'v1';
+
 before(function () {
-  if (!Cypress.env('ADMIN_TOKEN')) {
+  if (!Cypress.env('IAM_VERSION')) {
+    // tslint:disable-next-line:no-string-throw
+    throw('MISSING ENVIRONMENT VARIABLE: ' +
```

Comment:
@bcmdarroch cypress added the sarcastic smile, not me. If it's trying to communicate what you are suggesting that was completely lost on me LOL. Lemme give that a try.

Reply:
Looks much better with `throw new Error`. Good catch! Totally missed that that's what that cypress output was trying to communicate lol. The smilie threw me off:

```
Error: MISSING ENVIRONMENT VARIABLE: You must pass CYPRESS_IAM_VERSION. Must be one of: "v2.1", "v1".

Because this error occurred during a 'before all' hook we are skipping all of the remaining tests.
```

Reply:
Yeah not super helpful, cypress 😵